collector/lib/CollectorService.cpp

```diff
@@ -58,7 +58,9 @@ void CollectorService::RunForever() {
   prometheus::Exposer exposer("9090");
   exposer.RegisterCollectable(registry);
 
-  config_->Start();
+  if (config_->EnableRuntimeConfig()) {
```

Comment:
`EnableRuntimeConfig()` was originally put in place to start capturing namespace information from Falco. If this is no longer needed and/or the purpose of the configuration parameter has been changed, please adjust it accordingly, we don't want to have a performance impact for something we are not using.

Reply:
I think that is one of the purposes of `EnableRuntimeConfig()`. I think the main purpose is to control if runtime configuration is enabled or not. That is why it is called what it is and not `EnableNamespaceInfo()`. If runtime configuration is not used then there is no need to obtain namespace information. Also if runtime configuration is not used, the runtime configuration should not be obtained from the ConfigMap.